### PR TITLE
Added Backstage logo to the Open Graph Image metadata of backstage.io

### LINKED ISSUE #2054: Add Backstage logo to the Open Graph Image metadata of backstage.io
## Feature Suggestion

When we share https://backstage.io over social media like Twitter, it is good that the card they generate for the link has useful information.

Screenshot 2020-08-20 at 21 50 05

Currently Twitter does not get any image to use for backstage.io

## Possible Implementation

~Add an [`og:image`](https://ogp.me/) metadata to the website and use the backstage logo. I think we already have `og:title` and `og:description`, but I might be wrong.~

Use `png` instead of `svg` [here](https://github.com/spotify/backstage/blob/95596b812983f892411c32fa67a5ca7de4b22cd3/microsite/siteConfig.js#L99-101).

## Context

Use https://cards-dev.twitter.com/validator to validate results.

### PR DESCRIPTION
**Pull request by iamrajiv**
## Hey, I just made a Pull Request!

Fixes: #2054 

#### :heavy_check_mark: Checklist

- [x] All tests are passing `yarn test`
- [ ] Screenshots attached (for UI changes)
- [ ] Relevant documentation updated
- [x] Prettier run on changed files
- [ ] Tests added for new functionality
- [ ] Regression tests added for bug fixes

### CODE REVIEW

File: microsite/siteConfig.js [hunk 1]
```diff
@@ -93,8 +93,10 @@ const siteConfig = {
   cleanUrl: true,
 
   // Open Graph and Twitter card images.
-  ogImage: 'img/logo-gradient-on-dark.svg',
-  twitterImage: 'img/logo-gradient-on-dark.svg',
+  ogImage:
+    'logo_assets/png/Backstage Identity_Assets_Artwork_RGB_04 Icon_Teal.png',
```
Comment (freben) [inline microsite/siteConfig.js:97]: `%20` for the spaces perhaps? then again - i would really prefer if these filenames didn't have spaces in them to begin with 🙅

Comment (iamrajiv) [reply microsite/siteConfig.js]: @freben Yeah, I preferred the same. I think spaces should be removed from all the images filename. All the images from `logo_assets` have spaces in there filename.

File: microsite/siteConfig.js [hunk 2]
```diff
@@ -93,8 +93,10 @@ const siteConfig = {
   cleanUrl: true,
 
   // Open Graph and Twitter card images.
-  ogImage: 'img/logo-gradient-on-dark.svg',
-  twitterImage: 'img/logo-gradient-on-dark.svg',
+  ogImage:
+    'logo_assets/png/Backstage%20Identity_Assets_Artwork_RGB_04%20Icon_Teal.png',
```
[the remark this answers is not in the document]
Comment (stefanalund) [reply microsite/siteConfig.js:97]: +1

**Comment by OrkoHunter** [comment]
@iamrajiv Can you deploy your fork of backstage.io somewhere (GitHub pages)? We can test if it works by using https://cards-dev.twitter.com/validator to validate the results. :)

File: microsite/siteConfig.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (iamrajiv) [reply microsite/siteConfig.js:97]: @timja @stefanalund Should I make another PR to remove spaces from image name or should I do in this PR only.

**Comment by iamrajiv** [comment]
@OrkoHunter I have tried to deploy on GitHub pages but it is not working
Screenshot 2020-09-10 at 7 04 03 PM

**Comment by OrkoHunter** [comment]
Aah! Let me try to help. I believe there are some extra steps to serving this website.
1. Go to `microsite/` directory. https://github.com/spotify/backstage/tree/master/microsite
2. Build and start the website.

Now there are many things you can do here.
Easy: Try [`ngrok`](https://ngrok.com/) to generate a public URL for the website you have running locally and use the twitter validator on that public URL.
Easy: There is a chrome extension you can use. [this](https://chrome.google.com/webstore/detail/localhost-open-graph-chec/gcbnmkhkglonipggglncobhklaegphgn?hl=en)
Not Easy: Create a new branch, move the contents in the build branch and push. Now publish the new branch using GitHub pages.

**Comment by stefanalund** [comment]
@OrkoHunter we can also just get this merged and test it afterwards :)

**Comment by iamrajiv** [comment]
@OrkoHunter I have built the site and pushed it to my repository in a separate branch. I have checked https://cards-dev.twitter.com/validator its working fine. Site link https://iamrajiv.github.io/backstage/.

**Comment by OrkoHunter** [comment]
Hey @iamrajiv, Sorry I can't see the Backstage logo here, it's still missing. :(
Screenshot 2020-09-10 at 16 48 37

File: microsite/siteConfig.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (OrkoHunter) [reply microsite/siteConfig.js:97]: +1 You can rename the file in this PR and use the newer name. How about "Backstage_Icon_Teal.png"? 😄

**Comment by OrkoHunter** [comment]
@stefanalund We don't know how many PRs it might take if we test it after merging. 😅

**Comment by iamrajiv** [comment]
@OrkoHunter I am getting this.
Screenshot 2020-09-10 at 8 23 44 PM

**Review by OrkoHunter — APPROVED** [review]
Thanks @iamrajiv for working on this. LGTM 💯

**Comment by iamrajiv** [comment]
@OrkoHunter Thankyou @OrkoHunter @stefanalund for guidance.

**Review by stefanalund — APPROVED** [review]

**Comment by stefanalund** [comment]
Thanks a lot!

**Pull request merged**